src/styl/sections/index.styl

```diff
@@ -92,10 +96,12 @@
 .section_projects-container
   position relative
   width 100vw
+  padding-top remify(60)
```

Comment:
Don't use remify for padding, only use for attribute `font-size` https://davidwalsh.name/stylus-rem use % or rems native